pubspec.yaml

```diff
@@ -7,7 +7,7 @@ homepage: https://github.com/dart-lang/dartdoc
 environment:
   sdk: '>=1.14.0 <2.0.0'
 dependencies:
-  analyzer: 0.27.3-alpha.7
+  analyzer: 0.27.4-alpha.8
```

Comment:
That surprised me too.  (And partly what lulled me into thinking this alpha push was OK.)

To be sure we're on the same page, you are running `dartdoc` from the dart SDK?  And the latest dev build?